gen.sh

```diff
@@ -364,12 +368,15 @@ tc () {
           PARALLELISM_ACTIVE=1
           LN="cp "
         fi
-        local path="$CURGROUP_DIR/$(_base ${cases[$name]})"
+        local path
+        path="$CURGROUP_DIR/$(_base "${cases[$name]}")"
+        # shellcheck disable=2086
         ${LN}${cases[$name]}.in "$path.in"
```

Comment:
Quotes around `${cases[$name]}.in` would be fine here

Reply:
Added